### PR TITLE
Add updateLoanToken dftx

### PR DESCRIPTION
**Pull request by jingyi2811**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:

Add updateLoanToken dftx

#### Which issue(s) does this PR fixes?:

Fixes part of #532

### CODE REVIEW

File: packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_update_loan_token.test.ts [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+    })
+    await testing.generate(1)
+
+    await fundEllipticPair(testing.container, providers.ellipticPair, 10) // Fund 10 DFI UTXO
+    await providers.setupMocks() // Required to move utxos
+
+    const script = await providers.elliptic.script()
+    const txn = await builder.loans.updateLoanToken({
+      symbol: 'Token1',
+      name: 'Token1',
```
Comment (surangap) [inline packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_update_loan_token.test.ts:246]:
```suggestion
      name: 'Token5',
```
?

File: packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_update_loan_token.test.ts [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+      currencyPair: { token: 'Token1', currency: 'USD' },
+      mintable: true,
+      interest: new BigNumber(0.08),
+      tokenTx: loanTokenId
+    }, script) // Same name as Token1's name
+    await sendTransaction(testing.container, txn)
+
+    const data = await testing.container.call('listloantokens', [])
+    const index = Object.keys(data).indexOf(loanTokenId) + 1
+    expect(data[loanTokenId].token[index].name).toStrictEqual('Token1')
```
Comment (surangap) [inline packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_update_loan_token.test.ts:256]:
```suggestion
    expect(data[loanTokenId].token[index].name).toStrictEqual('Token5')
```
?

File: packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_update_loan_token.test.ts (hunk 1 above)
Comment (jingyi2811) [reply packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_update_loan_token.test.ts]: Fixed

File: packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_update_loan_token.test.ts (hunk 2 above)
Comment (jingyi2811) [reply packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_update_loan_token.test.ts]: Fixed

File: packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_update_loan_token.test.ts (hunk 1 above)
Comment (jingyi2811) [reply packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_update_loan_token.test.ts]: Actually `Token1` is correct. I change another token to have same name.

**Review by surangap — APPROVED** [review]

**Review by canonbrother — APPROVED** [review]

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**